src/input_streams.rs

```diff
@@ -266,14 +268,24 @@ impl<'a> InputStreams<'a> {
 
         // Helper that takes the value returned by an axis and returns 0.0 if it is not within the
         // triggering range.
-        let value_in_axis_range = |axis: &SingleAxis, value: f32| -> f32 {
-            if value >= axis.negative_low && value <= axis.positive_low && include_deadzone {
-                0.0
-            } else if axis.inverted {
-                -value * axis.sensitivity
-            } else {
-                value * axis.sensitivity
+        let value_in_axis_range = |axis: &SingleAxis, mut value: f32| -> f32 {
```

Comment:
This is a little gross but otherwise the single axis scaling would affect the dual axis scaling. Perhaps there is a better way I did not think about?